package.json

```diff
@@ -24,64 +43,37 @@
     "husky": "^3.1.0",
     "lint-staged": "^9.5.0",
     "lodash.throttle": "^4.1.1",
-    "nanoid": "^3.1.12",
+    "notistack": "^1.0.0",
     "prettier": "^1.19.1",
-    "react": "^16.12.0",
-    "react-dom": "^16.12.0",
     "react-router-dom": "^5.1.2",
-    "react-scripts": "^3.4.4",
     "strip-color": "^0.1.0",
-    "twilio": "^3.39.3",
-    "twilio-video": "2.8.0-beta.1",
-    "typescript": "^3.8.3"
-  },
-  "devDependencies": {
-    "mocha": "^6.2.2",
-    "react-test-renderer": "^16.12.0",
-    "start-server-and-test": "^1.10.8",
-    "ts-jest": "^24.3.0",
-    "puppeteer": "^5.3.1",
-    "mocha-junit-reporter": "^1.23.3",
-    "pixelmatch": "^5.1.0",
-    "isomorphic-fetch": "^2.2.1",
-    "jest-junit": "^9.0.0",
+    "@testing-library/jest-dom": "^4.2.4",
+    "@testing-library/react": "^9.4.0",
+    "@testing-library/react-hooks": "^3.2.1",
+    "@types/enzyme": "^3.10.4",
+    "@types/enzyme-adapter-react-16": "^1.0.5",
     "cypress": "^5.4.0",
     "cypress-multi-reporters": "^1.4.0",
     "enzyme": "^3.11.0",
     "enzyme-adapter-react-16": "^1.15.2",
     "enzyme-to-json": "^3.4.4",
-    "@testing-library/jest-dom": "^4.2.4",
-    "@testing-library/react": "^9.4.0",
-    "@testing-library/react-hooks": "^3.2.1",
-    "@types/enzyme": "^3.10.4",
-    "@types/enzyme-adapter-react-16": "^1.0.5"
-  },
-  "lint-staged": {
-    "src/**/*.{js,jsx,ts,tsx,json,css,scss,md}": [
-      "prettier --write",
-      "git add"
-    ]
-  },
-  "scripts": {
-    "start": "concurrently npm:server npm:dev",
-    "dev": "react-scripts start",
-    "build": "node ./scripts/build.js",
-    "test": "jest",
-    "eject": "react-scripts eject",
-    "lint": "eslint src/**/*.{ts,tsx}",
-    "server": "node server.js",
-    "test:ci": "jest --ci --runInBand --reporters=default --reporters=jest-junit --coverage",
-    "cypress:open": "cypress open",
```

Comment:
Do the cypress tests no longer work with the changes? Or can they be added back in later? I think it would be nice to have them.

Reply:
Yeah they still work, i just didn't want to force the web-app to install those cypress includes as a required dependency.